packages/schemas/tables/usersroles.sql

```diff
@@ -0,0 +1,5 @@
+create table users_roles (
```

Comment:
nit: why not rename this file as `users_roles.sql`? The current file name seems does not fit the naming convention. So do `scopesroles.sql`.

Reply:
Discussed offline.